lighthouse-core/test/audits/dobetterweb/js-libraries-test.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const JsLibrariesAudit = require('../../../audits/dobetterweb/js-libraries.js');
+const assert = require('assert');
+
+/* eslint-env jest */
+describe('Returns detected front-end JavaScript libraries', () => {
+  it('always passes', () => {
+    // no libraries
+    const auditResult1 = JsLibrariesAudit.audit({
+      JSLibraries: [],
+    });
+    assert.equal(auditResult1.rawValue, true);
+
+    // duplicates. TODO: consider failing in this case
```

Comment:
> TODO: consider failing in this case

it's not *that* uncommon to have more than one jquery in one page. And if we merge network stuff from iframes, won't it be even more common?